HesoyamHospital/GraphicEditor/Repository/GraphicRepository.cs

```diff
@@ -29,21 +29,16 @@ public List<GraphicalObject> ReadFromFile(string fileName)
                         list.Add(graphical_object);
                 }
             }
-            else
-            {
-                return list;
-            }
+            
             return list;
         }
 
         private GraphicalObject ConvertLineToGraphicalObject(string line, string hospital, string floor)
         {
             string[] fields = line.Split(',');
-            if (fields.Length != 8)
-            {
 
-                return null;
-            }
+            if (fields.Length != 8) return null;
```

Comment:
Try throwing custom exceptions before or instead of returning null, so that the developer thats debugging the code could know if a possible null pointer exception came from here.